spec/function.dd

```diff
@@ -1899,6 +1899,38 @@ class C
         $(DDSUBLINK spec/expression, function_literals, lambdas) can deduce
         the `return scope` attribute.)
 
+$(H4 $(LNAME2 return-parameters, Return Parameters))
+
+        $(P A parameter with the `return` attribute but not `scope` means
+        its return value must not outlive the parameter when the parameter
+        is allocated on the stack.
+        )
+
+$(SPEC_RUNNABLE_EXAMPLE_COMPILE
+---
+@safe:
+
+struct S
+{
+    int* p;
+    // return applies to the `this` reference
+    int* getReturn() return => p;
+    int* getReturnScope() return scope => p;
+}
```

Comment:
There is no such thing as a "`return` parameter". In this example, the `this` reference passed to `getReturn` is a `return ref` parameter, because the `this` parameter of a `struct` method is implicitly `ref`.

Reply:
OK, thanks. Updated.